docs/package_firewall/cargo.md

```diff
@@ -0,0 +1,116 @@
+# Cargo Package Firewall
+
+The Cargo package firewall is based on Cargo's sparse [Registry Index API].
+
+[Registry Index API]: https://doc.rust-lang.org/cargo/reference/registry-index.html
+
+## Configuration
+
+All configuration options will require a Phylum API key, since Phylum's package
+firewall requires authentication. You can find out how to generate one in our
```

Comment:
nit: I think since clauses are usually found at the beginning of a sentence.
```suggestion
Since Phylum's package firewall requires authentication, all configuration
options require a Phylum API key. You can find out how to generate one in our
```

Reply:
I'm not a native speaker, but I prefer the current version and it is already present this way in the other ecosystems. It's better to put the important stuff first, so people have an easier time skipping over irrelevant paragraphs.